### ISSUE TITLE
Enable GitHub workflows for PRs with the ok-to-test label

### ISSUE THREAD

**Issue by ivanvc** [etcd-io/raft#116]
Allow to run GitHub workflows for Pull Requests with the `ok-to-test` label using the prow bot command /ok-to-test. This was already done in the etcd repo (refer to the issue https://github.com/etcd-io/etcd/issues/16956 and its PR https://github.com/etcd-io/etcd/pull/16965).

**Comment by yashkundu** [comment]
I want to do it, @jmhbnz , can you please assign this to me?

**Comment by jmhbnz** [comment]
Certainly!  Please reach out to @ivanvc or I if you have any questions 🙏🏻 

/assign @yashkundu

**Comment by yashkundu** [comment]
I have implemented it on [#117](https://github.com/etcd-io/raft/pull/117).
